src/Resources/contao/widgets/Password.php

```diff
@@ -114,9 +114,12 @@ protected function validator($varInput)
 			return '*****';
 		}
 
-		if (Utf8::strlen($varInput) < \Config::get('minPasswordLength'))
+		// Check password length either from DCA or use Config as fallback (#1086)
+		$intLength = ($this->arrConfiguration['minlength'] > 0) ? $this->arrConfiguration['minlength'] : \Config::get('minPasswordLength');
```

Comment:
You can skip the `$this->arrConfiguration` array and just use `$this->minlength`, it will use the magic `__get()` method. Not saying this is better but we have to make sure it's in line with all the other widgets :) 
Otherwise this looks good to me, it's nice to see more developers trying to contribute! Excellent!

Reply:
You are right, thanks!